package.json

```diff
@@ -26,7 +27,8 @@
     "build": "react-scripts test --watchAll=false --env=jsdom; react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "lint": "eslint --ext .jsx,.js src"
+    "lint": "eslint --ext .jsx,.js src",
+    "lint-fix": "eslint --fix --ext .jsx,.js src"
```

Comment:
perhaps rename this to `yarn style`, `style` being in verb form as 'resolving style issues'